src/Engine/Graphics/Outdoor.cpp

```diff
@@ -462,6 +462,19 @@ void OutdoorLocation::Release() {
     viewparams->location_minimap = nullptr;
 }
 
+void OutdoorLocation::fixResourceIssues() {
+    if (engine->_currentLoadedMapId == MAP_BARROW_DOWNS)
+    {
+        if (!pLevelDecorations.empty()) {
```

Comment:
Why would they be empty here?

Reply:
went overboard with safety - even if it was empty - it wouldn't matter

Reply:
this code is also no there any more